### ISSUE TITLE
No error reported for line length or todo

### ISSUE THREAD

**Issue by austinsmorris** [rrrene/credo#135]
No errors are reported for this file:

```elixir
defmodule Foo do
  @moduledoc false

  # todo - make this cool!
  def my_function(arg1, arg2, whatever) when whatever >= -9_223_372_036_854_775_808 and whatever <= 9_223_372_036_854_775_807, do: whatever
end
```

It seems like there should be...

**Comment by rrrene** [comment]
Function definitions are excluded from the line length check by default.

You can include them by using

```
{Credo.Check.Readability.MaxLineLength, ignore_definitions: false},
```

Hope this helps!

**Comment by austinsmorris** [comment]
That did it, thanks!

**Comment by austinsmorris** [comment]
Also, I created #138, about the todo check...
